Dockerfile

```
@@ -27,7 +27,7 @@ RUN addgroup \
     && mkdir --parents ${CONTAINER_HOME} \
     && chown --recursive ${CONTAINER_USER}:${CONTAINER_GROUP} ${CONTAINER_HOME} \
     && apk add --no-cache --virtual build \
-      curl==8.7.1-r0 \
+      curl=~8 \
```

```suggestion
      curl==8.8.0-r0 \
```

as per https://pkgs.alpinelinux.org/packages?name=curl&branch=v3.20&repo=&arch=&maintainer=